### ISSUE TITLE
MacOS version

### ISSUE THREAD

**Issue by kutuz00** [SoftProjector/softprojector#28]
Do you plan to make release for MacOS users?

**Comment by senechal-ua** [comment]
Yes, release for Mac is on official site http://softprojector.org/download.html

There are bugs when playing video. This should be solved after replacing the video player

**Comment by kutuz00** [comment]
I am talking about softprojector 2.2

**Comment by SlavikCA** [comment]
Any update on version 2.2?
